packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -296,7 +297,8 @@ storiesOf('ListView/Actions', module)
           </Item>
         </ActionMenu>
       </>
-    ), args));
+    ), args))
+  .add('Restore focus after item removal', (args = {}) => <FocusExample {...args} />);
```

Comment:
Mind adding one more parameter to line 66? Something like this:
```
    disabledBehavior: {
      control: {
        type: 'radio',
        options: ['selection', 'all']
      }
    }
```
a default can be provided to line 39 via `disabledBehavior: 'selection'`. I just realized I mistakenly conflated `highlight selection` and `disabledBehavior` when testing yesterday